### ISSUE TITLE
Return typescript types back

### ISSUE THREAD

**Issue by aleksandrlat** [DataDog/browser-sdk#654]
Some types and enums were removed 
https://github.com/DataDog/browser-sdk/pull/630/files#diff-0a7a0b18b6e69b1211c9c7ae7d3c838a36d248db33f06f4c6cb997818ad7aba6

We need to duplicate some types now. Please return them back.

**Comment by aleksandrlat** [comment]
Can/should we use something like this now?
```
import { StatusType } from '@datadog/browser-logs/esm/domain/logger'
```

**Comment by bcaudan** [comment]
Hi @aleksandrlat, 

Our intention was to facilitate the use of our APIs by replacing enum type by string union and enum value by string.
before: 
```ts
import { StatusType, datadogLogs } from '@datadog/browser-logs'

datadogLogs.logger.log('message', {}, StatusType.error)
```
after: 
```ts
import { datadogLogs } from '@datadog/browser-logs'

datadogLogs.logger.log('message', {}, 'error')
```

We could still export the values if it is helpful though.
Would that suit your usage or is there another issue?

**Comment by aleksandrlat** [comment]
Thank you @bcaudan!

We have our wrapper around your lib. And it is helpful to use these types in wrapper or code related to it.

**Comment by bcaudan** [comment]
Oh interesting, would you mind sharing the purpose of the wrapper and how you wrap the lib?

**Comment by aleksandrlat** [comment]
@bcaudan first of all we have our logger and datadog is only part of our logger. We abstract away datadog by our logger.
Second we have throttling in logger and group same errors into one.

**Comment by BenoitZugmeyer** [comment]
This issue is addressed by PR #662 and will be released shortly.
